.asset-template.json.hb

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Block Coding",
+  "description": "Create games using a high-level, block-based visual programming language.\r\n\r\nIntended as an educational tool for learners in the earlier stages of their journey towards becoming game developers. This plugin lets you create your first games with high-level blocks, avoiding the immediate need to learn to code in GDScript. Building games in this way provides a gentle introduction to programming concepts and allows you to focus your efforts on becoming familiar with the rest of the Godot Editor UI.",
+  "category_id": "5",
+  "godot_version": "4.2",
+  "version_string": "{{ context.release.tag_name }}",
+  "cost": "MIT",
+  "support_level": "community",
+  "download_provider": "GitHub",
+  "download_commit": "{{ env.GITHUB_SHA }}",
+  "browse_url": "{{ context.repository.html_url }}",
+  "issues_url": "{{ context.repository.html_url }}/issues",
+  "icon_url": "https://raw.githubusercontent.com/endlessm/godot-block-coding/{{ context.release.tag_name }}/icon.png",
+  "download_url": "https://github.com/endlessm/godot-block-coding/archive/99f8587ad7bcc914ab6bf8bb1e11665f91a73ca8.zip"
```

Comment:
That corresponds to the v0.1.0 release. Do we need both `download_commit` and `download_url`? If `download_url` is needed, I'm pretty sure you could use `https://github.com/endlessm/godot-block-coding/archive/refs/tags/{{ context.release.tag_name }}.zip`. Also, if I'm reading [the release payload correctly](https://docs.github.com/en/webhooks/webhook-events-and-payloads#release), you could use `{{ context.release.zipball_url }}`.

Reply:
I think `download_url` is needed as that appears to be exactly what's put in the Download button on the Asset Library page.

Reply:
Actually I don't think we need `download_url`, good point.

Reply:
It is constructed server-side from the GitHub browse URL and commit.

Reply:
Removed `download_url`, following the example at https://github.com/deep-entertainment/godot-asset-lib-action?tab=readme-ov-file#example-template